docs/conf.py

```diff
@@ -698,6 +701,7 @@ def _get_params(root_schema: dict, prefix: str = "", default_section: str = "")
     autoapi_ignore.append('*/airflow/providers/*')
 else:
     autoapi_ignore.append('*/airflow/providers/cncf/kubernetes/backcompat/*')
+    autoapi_ignore.append('example_dags/*')
```

Comment:
Example dags were moved to point at github (and excluded in output), but we were still parsing them for autoapi